src/css/main.less

```
@@ -1715,19 +1715,33 @@ dialog {
 		height: auto;
 	}
 }
-.fixtrue {
-	background-color: #56ac1d;
-	padding: 2px 5px;
-	border-radius: 3px;
-	color: #fff;
-	font-size: 10px;
-}
-.fixfalse {
-	background-color: #e60000;
-	padding: 2px 5px;
-	border-radius: 3px;
-	color: #fff;
-	font-size: 10px;
+.fix {
+    background-color: #e60000;
+    padding: 2px 5px;
+    border-radius: 3px;
+    color: #fff;
+    font-size: 10px;
+}
+.fix.active {
+    background-color: #56ac1d;
+    padding: 2px 5px;
+    border-radius: 3px;
+    color: #fff;
+    font-size: 10px;
+}
+.vtx_device_ready {
+    background-color: #e60000;
+    padding: 2px 5px;
+    border-radius: 3px;
+    color: #fff;
+    font-size: 10px;
+}
+.vtx_device_ready.active {
+    background-color: #56ac1d;
+    padding: 2px 5px;
+    border-radius: 3px;
+    color: #fff;
+    font-size: 10px;
```

I've not tested it, but what I wanted to say is to use something like this:
```suggestion
.fix {
    background-color: #e60000;
    padding: 2px 5px;
    border-radius: 3px;
    color: #fff;
    font-size: 10px;

    &.active {
        background-color: #56ac1d;
    }
}
.vtx_device_ready {
    background-color: #e60000;
    padding: 2px 5px;
    border-radius: 3px;
    color: #fff;
    font-size: 10px;
    
    &.active {
        background-color: #56ac1d;
    }
}
```
But, two suggestions more:
- Don't call `active` the css class, it can be confuse because the pseudo-class `:active` exists. Maybe something like `positive` is better.
- The `fix` and `vtx_device_ready` class are exactly the same. Why don't we use one for both? Being in the `main.less` class, something generic that can be used at each place where we want to use it.

---

good suggestion, changed to this
```
.colorToggle {
    background-color: #e60000;
    padding: 2px 5px;
    border-radius: 3px;
    color: #fff;
    font-size: 10px;
	&.ready {
		background-color: #56ac1d;
	}
}
```
and
```
        const vtxReady_e = $('.VTX_info span.colorToggle');
        vtxReady_e.text((FC.VTX_CONFIG.vtx_device_ready) ? i18n.getMessage('vtxReadyTrue') : i18n.getMessage('vtxReadyFalse'));
        vtxReady_e.toggleClass('ready', FC.VTX_CONFIG.vtx_device_ready);
```